database/manual-updates/2022-03-16_2-add-new-fields-to-allocations-table-postgres.md

```diff
@@ -0,0 +1,25 @@
+# Additional fields for Allocation
+
+## The problem we're trying to solve
+
+We need to add RAG rating, Summary and CarePackage fields to the allocations table.
+## Justification for doing a manual update
+
+We don't have automated db migrations
+
+## The plan
+
+1. Run the SQL query
+
+## Link to Jira ticket
+
+[SCT-1736] (https://hackney.atlassian.net/browse/SCT-1758)
+
+## SQL statement(s)
+
+```sql
+alter table if exists dbo.SCCV_ALLOCATIONS_COMBINED
+    add column RAG_RATING varchar,
```

Comment:
would be a good practise to set some sensible column sizes for string values rather that opting for max size by default

Reply:
We are not sure yet what will be final rating exaclty, so no reserve at this point.